moped-database/metadata/tables.yaml

```diff
@@ -4924,30 +4925,31 @@
           - project_sponsor
           - project_tags
           - project_team_members
+          - public_process_status
```

Comment:
this is mostly cleanup except for adding permissions for this field for the different roles